### ISSUE TITLE
po2 data distribution in circulatory/resoiratory/digestive patients

### ISSUE THREAD

**Issue by shoyone** [MIT-LCP/mimic-code#218]
Hi Support team,
I just would like to verify historical data distribution of “po2”.
Now I try to interpret po2 distribution in MIMIC.

When I filter the po2 data as below, the values in first 5 hours from ICU admission seem higher than I thought. 

Here is the filter conditions of patients to use po2 values:
- Age :older than 20
- diagnosis_cd :
ICD9 390–459 diseases of the circulatory system
or 460–519 diseases of the respiratory system
or 520–579 (excl.430-438) diseases of the digestive system
- Admission type :Elective

In addition, below is the line chart of po2
(x: time from admission, y: po2, one line: one patient)

<img width="646" alt="2017-05-24 8 56 37" src="https://cloud.githubusercontent.com/assets/25365228/26381331/f60d7572-405e-11e7-882a-3f46247ec92c.png">

When we see circle green, the values look higher than those after 5 hours from admission. 

I was wondering if you know the reason why first 5 hours numbers only are so high.
I understand sometimes po2 could take these values(400 or 500 mmhg).
I just would like to know there is some specific reasons to be like this if you know.

I would appreciate it if you would give me some advice.
Thank you for any help you can provide.

**Comment by chris20881** [comment]
One possibility - people admitted to ICU are often initially on more oxygen than they would need to achieve a 'normal' pO2. This could be for a number of reasons, resuscitation protocols may call for a high percentage of oxygen to be administered, or unstable patients may be given a high percentage to provide some room for a fall. Even stable elective patients brought from an operation may be put on more than they 'need' for the transfer, for safety reasons. These higher than needed FiO2s could then lead to the high pO2s. After a few hours the patients are reassessed and the FiO2 lowered to what they actually need to achieve an adequate pO2.
(I've not posted comments here before - and sorry, have no idea how many looking at this have clinical background or what might be useful to comment on, or what depth. Please do advise if this is not useful / appropriate).

**Comment by alistairewj** [comment]
These kind of comments are more than welcome! Not everyone has a clinical background so the feedback is very useful. I think your explanation is a pretty sensible reason for why this occurs. We actually noticed this phenomenon this week, and when we plotted the maximum pO2 among non-ventilated patients, the large values no longer occurred. I think this supports your hypothesis that it is related to resuscitation protocols and/or safeguarding the patient against falls, perhaps prior to extubation.
